src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -92,6 +93,23 @@ public static FunctionPointerMethodSymbol CreateFromSource(FunctionPointerTypeSy
                 suppressUseSiteDiagnostics);
         }
 
+        internal static FunctionPointerMethodSymbol CreateFromParts(
```

Comment:
>CreateFromParts [](start = 52, length = 15)

Consider adding a comment that this helper can be used only in scenarios that are not interested in reporting any diagnostics. And probably the same comment for the new constructor.